server/options.go

```diff
@@ -21,12 +21,18 @@
 package xserver
 
 import (
+	"time"
+
 	"github.com/m3db/m3x/instrument"
 	"github.com/m3db/m3x/retry"
 )
 
 const (
+	// By default keepAlives are enabled for TCP connections.
 	defaultTCPConnectionKeepAlive = true
+
+	// By default the keep alive period is not set.
```

Comment:
also indicate that the default keep alive period in tcpConn will be used when no override value is provided?